packages/replay/src/util/addEvent.ts

```diff
@@ -46,5 +47,12 @@ export async function addEvent(
   } catch (error) {
     __DEBUG_BUILD__ && logger.error(error);
     replay.stop();
+
+    const hub = getCurrentHub();
+    const client = hub.getClient();
```

Comment:
super-l: Likewise here
```suggestion
    const client = getCurrentHub().getClient();
```
?